<launch>
    <arg name="image_topic" default="/xtion/rgb/image_raw"/>
    <arg name="depth_topic" default="/xtion/depth_registered/image_raw"/>
    <arg name="info_topic" default="/xtion/depth_registered/camera_info"/>

    <node pkg="clf_object_recognition_3d" type="simple_3d" name="yolox_detection_3d">
        <param name="image_topic" default="$(arg image_topic)"/>
        <param name="depth_topic" default="$(arg depth_topic)"/>
        <param name="info_topic" default="$(arg info_topic)"/>
        <param name="detect_2d_topic" value="/yolox/recognize"/>
	</node>

</launch> 